lldb/tools/debugserver/source/RNBRemote.cpp

```diff
@@ -3594,6 +3594,9 @@ rnb_err_t RNBRemote::HandlePacket_qSupported(const char *p) {
   reply << "SupportedWatchpointTypes=x86_64;";
 #endif
 
+  // We consume lldb's swbreak/hwbreak feature, but it doesn't change the
+  // behaviour of lldb-server. We always adjust the program counter for targets
+  // like x86
```

Comment:
Sorry I missed that this is in `debugserver` not `lldb-server`. Confusing I know, but Apple platforms use debugserver which shares some common code.

This comment should go in `GDBRemoteCommunicationServerLLGS::HandleFeature` instead, just after the `// check for client features` for loop should be fine.

Reply:
Now fixed in https://github.com/llvm/llvm-project/commit/51a356b384f28cce3b2cd874fbba139414b0e207